esp/src/src-react/components/DFUWorkunits.tsx

```diff
@@ -124,15 +126,32 @@ export const DFUWorkunits: React.FunctionComponent<DFUWorkunitsProps> = ({
                 }, [])
             },
             User: { label: nlsHPCC.Owner, width: 90 },
-            JobName: { label: nlsHPCC.JobName, width: 500 },
-            ClusterName: { label: nlsHPCC.Cluster, width: 126 },
-            StateMessage: { label: nlsHPCC.State, width: 72 },
+            JobName: { label: nlsHPCC.JobName, width: 220 },
+            ClusterName: { label: nlsHPCC.Cluster, width: 70 },
+            StateMessage: { label: nlsHPCC.State, width: 70 },
             PCTDone: {
-                label: nlsHPCC.PctComplete, width: 90, sortable: true,
+                label: nlsHPCC.PctComplete, width: 80, sortable: true,
                 formatter: React.useCallback(function (value, row) {
                     return Utility.valueCleanUp(row.PercentDone);
                 }, [])
+            },
+            TimeStarted: { label: nlsHPCC.TimeStarted, width: 100, sortable: true },
+            TimeStopped: { label: nlsHPCC.TimeStopped, width: 100, sortable: true },
+            KbPerSec: {
+                label: nlsHPCC.TransferRate, width: 90,
+                formatter: React.useCallback(function (value, row) {
+                    return Utility.convertedSize(row.KbPerSec * 1024) + " / sec";
+                }, [])
+            },
+            /*
+            // KbPerSecAve seems to never be different than KbPerSec at the moment, see HPCC-29894
+            KbPerSecAve: {
+                label: nlsHPCC.TransferRateAvg, width: 90,
+                formatter: React.useCallback(function (value, row) {
+                    return Utility.convertedSize(row.KbPerSecAve * 1024) + " / sec";
+                }, [])
             }
+            */
```

Comment:
Commented code?

Reply:
Yeah, the KbPerSecAve column should be displayed as well, since it was part of Kevin's ESP change for this. But I noticed currently it isn't ever different from the KbPerSec value that comes back. So I opened that [29894 ](https://track.hpccsystems.com/browse/HPCC-29894) issue.